docs/LIT.md

```diff
@@ -0,0 +1,168 @@
+# LLVM Integrated Tester (LIT) in Glow
+
+Glow supports LLVM style LIT tests. These tests provide a way to run our high
+level tools through command line invocations like a user would do.
+
+To run these test, glow needs to know where to find two tools:
+* `lit`: This is the driver of the lit testsuite.
+* `FileCheck`: This is the tool used to check that the test ran correctly.
+
+To use the LIT testing, you need to have both these tools in your path before
+invoking CMake. More specifically, CMake will determine whether to enable the
+LIT testing based on the presence or absent of these two executables.
+
+Note: `lit` and `lit.py` can be used indistinctly, CMake will recognize both.
+
+When those tools are set, a typical CMake invocation supporting our LIT tests
+looks like this:
+```bash
+ cmake -G Ninja <glow_src>  -DCMAKE_BUILD_TYPE=Release \
+       -DCMAKE_PREFIX_PATH=<path_to_llvm_install>      \
+       -DGLOW_MODELS_DIR=<downloaded_c2_models>
+```
+
+## Installing the Tools
+
+### From LLVM source
+
+Both tools `lit.py` and `FileCheck` are available directly from the LLVM source and our
+`utils/build_llvm.sh` makes them readily available.
+
+Alternatively, `utils/set_llvm_test_env.sh` provides a light way of exporting
+the tools while not building LLVM in full.
+To use it, simply run:
+```bash
+source utils/set_llvm_test_en.sh
+```
+This will build the required tools and update your `PATH` accordingly.
+
+### Using a Package Manager
+
+LIT is a python package that can easily be obtained through a python package
+manager.
+Simply run:
+```bash
+pip install lit
+```
+
+If using `brew`, `FileCheck` comes with the LLVM package that we already described in the earlier section. I.e., `FileCheck` comes with:
```

Comment:
format to 80col

Reply:
Nice catch!
I thought I formatted all of these but turns out I didn't!!
Should be fixed now.